spotify/views/albums.hbs

```diff
@@ -0,0 +1,13 @@
+<link rel="stylesheet" href="/stylesheets/albums.css">
+
+<h1>Musica</h1>
+
+<div class="albums-container">
+  {{#each albums}}
+  <div class="albums">
+    <img src="{{this.images.0.url}}" height="300" alt="{{this.name}}">
```

Comment:
Aquí es posible que se imprima imágenes rotas (si el album no tiene), para evitar eso podéis hacer un check de si el array está vacío: 
{{#if this.images.length}} la imagen {{/if}}